src/imitation/scripts/common/seeding.py

```diff
@@ -0,0 +1,26 @@
+"""Single source of truth for seeding random number generators in experiments."""
```

Comment:
I was going to do that, but that forces other ingredients and experiments to depend on common, and thought it might not be desirable. Since you're okay with it, I'll change it.